### ISSUE TITLE
Accept trade offer issue

### ISSUE THREAD

**Issue by am576** [bukson/steampy#346]
Was trying this library but it seems that i can't accept offers for some reasons, i get
```
File "\venv\lib\site-packages\steampy\client.py", line 256, in accept_trade_offer
    partner = self._fetch_trade_partner_id(trade_offer_id)
  File "\venv\lib\site-packages\steampy\client.py", line 282, in _fetch_trade_partner_id
    return text_between(offer_response_text, "var g_ulTradePartnerSteamID = '", "';")
  File "\venv\lib\site-packages\steampy\utils.py", line 29, in text_between
    start = text.index(begin) + len(begin)
ValueError: substring not found
```
when trying to `steam_client.accept_trade_offer(offer_id)`
Though I follow the  [example presented here](https://github.com/bukson/steampy/blob/master/examples/storehouse.py)
There was a simiar issue a while ago, but it appeared the issue was due to dead session, but I call `steam_client.login()` just before accepting trades

**Comment by lspeed-v** [comment]
The library is broken because of steam update, look previous issue discussion. We should w8 when @bukson will fix thix lib.

**Comment by am576** [comment]
> The library is broken right now because of steam update, look at previous issue discussion. We should w8 when @bukson will fix this lib. Btw here's manual fix for that [#343 (comment)](https://github.com/bukson/steampy/issues/343#issuecomment-1901125725)

Thanks, seems to work now
